packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -74,6 +74,7 @@ class InteractiveViewer extends StatefulWidget {
     // use cases.
     this.maxScale = 2.5,
     this.minScale = 0.8,
+    this.scrollFriction = 0.0000135,
```

Comment:
```suggestion
    this.scrollFrictionCoefficient= 0.0000135,
```
Adding this suggestion in case you agree with https://github.com/flutter/flutter/pull/109443/files#r944361349, in this case you can use GitHub to batch the suggestions and automatically make the change.